oidc-wp/openid-connect-generic
# chore(Unit Testing): Adds Unit Testing & New Local Development Environment

### All Submissions:






### Changes proposed in this Pull Request:



- Refactors and adds unit tests.
- Fixes Gruntfile testing.
- Adds unit testing w/ Docker using the Dev Container image.
- Updates NPM & Composer dependencies.

### How to test the changes in this Pull Request:

1. Use either GitHub Codespaces or local Visual Studio Code. (see: https://github.com/forumone/wp-cfm/wiki/How-to-setup-the-plugin-development-environment)
2. Confirm that the development environment starts up.
3. Run `composer phpunit` to confirm running tests. (Also observe GitHub Actions)

### Other information:
